primitives/src/migrate.rs

```diff
@@ -165,3 +166,14 @@ where
         put_storage_value(module, item, &key, value);
     }
 }
+
+/// Kill a storage item from a module
+pub fn kill_item<H: StorageHasher>(module: &[u8], item: &[u8]) {
+    let module = H::hash(module);
+    let item = H::hash(item);
+    //TODO replace with [u8; N] when const generics lands
+    let mut prefix = Vec::with_capacity(module.as_ref().len() + item.as_ref().len());
+    prefix.extend_from_slice(module.as_ref());
+    prefix.extend_from_slice(item.as_ref());
+    kill_prefix(&prefix)
```

Comment:
Only `Twox128` is used for hashing the module and object name. Therefore, you don't need to enter the generic land. You can simply do
```
let mut final_key = [0u8; 32];	
final_key[0..16].copy_from_slice(&Twox128::hash(module));	
final_key[16..32].copy_from_slice(&Twox128::hash(item));
kill_prefix(&prefix)
```